packages/application-extension/src/index.ts

```diff
@@ -200,11 +200,18 @@ const opener: JupyterFrontEndPlugin<void> = {
   ): void => {
     const { commands, docRegistry } = app;
 
+    const ignoreTreePattern = new RegExp('/tree/(.*)');
```

Comment:
Curious if this approach works when a user creates a folder called "tree"?

Reply:
@jtpio Thanks for pointing this out, the above code will not work for the scenario that you've mentioned. I have revised the code with a new approach. Please have a look into it.